packages/paste-core/components/button/__tests__/button.test.tsx

```diff
@@ -8,375 +8,548 @@ import {shallow, mount} from 'enzyme';
-    wrapper.simulate('blur');
-    expect(onBlurMock).toHaveBeenCalledTimes(1);
+  describe.only('button default styles', () => {
+    it('should have the correct styles for the primary variant', () => {
+      const {getByTestId, getByText} = testRender(
+        <Theme.Provider theme="default">
+          <Button variant="primary" data-testid="primary-styles">
+            Primary
+          </Button>
+        </Theme.Provider>
+      );
+
+      const button = getByTestId('primary-styles');
+
+      expect(button).not.toHaveStyleRule('text-align', 'left');
+      expect(button).toHaveStyleRule('color', 'rgb(255,255,255)');
+      expect(button).toHaveStyleRule('background-color', 'rgb(2,99,224)');
+      expect(button).toHaveStyleRule('box-shadow', '0 0 0 1px #0263e0');
+
+      expect(getByText('Primary')).toHaveStyleRule('justify-content', 'center');
+    });
+
+    it('should have the correct styles for the secondary variant', () => {
+      const {getByTestId, getByText} = testRender(
+        <Theme.Provider theme="default">
+          <Button variant="secondary" data-testid="secondary-styles">
+            Secondary
+          </Button>
+        </Theme.Provider>
+      );
+
+      const button = getByTestId('secondary-styles');
+
+      expect(button).not.toHaveStyleRule('text-align', 'left');
+      expect(button).toHaveStyleRule('color', 'rgb(18,28,45)');
+      expect(button).toHaveStyleRule('background-color', 'rgb(255,255,255)');
+      expect(button).toHaveStyleRule('box-shadow', '0 0 0 1px #8891aa');
+
+      expect(getByText('Secondary')).toHaveStyleRule('justify-content', 'center');
+    });
+
+    it('should have the correct styles for the destructive variant', () => {
+      const {getByTestId, getByText} = testRender(
+        <Theme.Provider theme="default">
+          <Button variant="destructive" data-testid="destructive-styles">
+            Destructive
+          </Button>
+        </Theme.Provider>
+      );
+
+      const button = getByTestId('destructive-styles');
+
+      expect(button).not.toHaveStyleRule('text-align', 'left');
+      expect(button).toHaveStyleRule('color', 'rgb(255,255,255)');
+      expect(button).toHaveStyleRule('background-color', 'rgb(214,31,31)');
+      expect(button).toHaveStyleRule('box-shadow', '0 0 0 1px #d61f1f');
+
+      expect(getByText('Destructive')).toHaveStyleRule('justify-content', 'center');
+    });
+
+    it('should have the correct styles for the destructive_secondary variant', () => {
+      const {getByTestId, getByText} = testRender(
+        <Theme.Provider theme="default">
+          <Button variant="destructive_secondary" data-testid="destructive_secondary-styles">
+            Destructive secondary
+          </Button>
+        </Theme.Provider>
+      );
+
+      const button = getByTestId('destructive_secondary-styles');
+      expect(button).not.toHaveStyleRule('text-align', 'left');
+      expect(button).toHaveStyleRule('color', 'rgb(214,31,31)');
```

Comment:
I'm not too sure how valuable some of these are, mainly the color styles. Won't VRT in the stories catch changes here?  

As an aside, if we wanted to guarantee we always used the same token (rather than mapping the value), we can remove the `Theme.Provider` wrapper and check the token directly.

Reply:
Yes that makes total sense! I did that previously, but saw some test suites that are not checking the token values, but the actual css values. Will make that change now. Thanks!